proj/src/main/webapp/chat.js

```diff
@@ -557,8 +557,12 @@ function friendRequestButton(uid) {
 
                     const currFriendRef = firebase.database().ref('/users/'+currentUid+'/friends/'+uid);
                     const otherFriendRef = firebase.database().ref('/users/'+uid+'/friends/'+currentUid);
-                    currFriendRef.set(true);
-                    otherFriendRef.set(true);
+
+                    const oneOnOneRef = firebase.database().ref('/chat/chats-1on1');
```

Comment:
`chats-1on1` should probably be a constant in case it needs to be referenced in multiple places. Like how it is used again in `addFriendToDom`